### PR TITLE
azpainter: update to 3.0.6

### PR DESCRIPTION
**Pull request by hervyqa**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

**Comment by hervyqa** [comment]
```
=> ERROR: azpainter-3.0.6_1: PKGDESTDIR is empty and build_style != meta
=> ERROR: azpainter-3.0.6_1: cannot continue with installation!
```
`PKGDESTDIR is empty and build_style != meta` weird, i don't know why.

**Comment by kruceter** [comment]
Here you go.

```diff
diff --git a/srcpkgs/azpainter/patches/0001-fix-configure-error.patch b/srcpkgs/azpainter/patches/0001-fix-configure-error.patch
deleted file mode 100644
index 25c9483144..0000000000
--- a/srcpkgs/azpainter/patches/0001-fix-configure-error.patch
+++ /dev/null
@@ -1,39 +0,0 @@
-From 460cd59bcc0735ca4e1f4e14b59f4e3459c2044d Mon Sep 17 00:00:00 2001
-From: Nathan Owens <ndowens@artixlinux.org>
-Date: Mon, 30 Nov 2020 09:44:57 -0600
-Subject: [PATCH] fix configure error
-
----
- configure | 3 ---
- 1 file changed, 3 deletions(-)
-
-diff --git configure configure
-index cb9a9f6..06e65e0 100755
---- a/configure
-+++ b/configure
-@@ -203,9 +203,6 @@ for opt;do
- 		cf_check_lib=no ;;
- 	*=*)
- 		eval "$opt" ;;
--	*)
--		echo "unknown option: $opt"
--		exit 1 ;;
- 	esac
- done
- 
--- 
-2.29.2
-
-diff --git configure configure
-index 06e65e0..b10aed0 100755
---- a/configure
-+++ b/configure
-@@ -201,8 +201,6 @@ for opt;do
- 		cf_docdir=$tmp1 ;;
- 	--no-check-lib)
- 		cf_check_lib=no ;;
--	*=*)
--		eval "$opt" ;;
- 	esac
- done
- 
diff --git a/srcpkgs/azpainter/template b/srcpkgs/azpainter/template
index 05bd1be327..bd7441790a 100644
--- a/srcpkgs/azpainter/template
+++ b/srcpkgs/azpainter/template
@@ -1,15 +1,19 @@
 # Template file for 'azpainter'
 pkgname=azpainter
-version=2.1.6
+version=3.0.6
 revision=1
-build_style=gnu-configure
-hostmakedepends="libpng-devel libjpeg-turbo-devel libXi-devel
- fontconfig-devel"
-makedepends="$hostmakedepends"
+build_style=cmake
+hostmakedepends="pkg-config"
+makedepends="libpng-devel libjpeg-turbo-devel libwebp-devel libXi-devel fontconfig-devel
+ libXcursor-devel libXrender-devel tiff-devel"
 depends="desktop-file-utils hicolor-icon-theme"
 short_desc="Full-color painting software for illustration drawing"
 maintainer="Alexander Mamay <alexander@mamay.su>"
 license="GPL-3.0-or-later"
-homepage="https://github.com/Symbian9/azpainter"
-distfiles="https://github.com/Symbian9/azpainter/releases/download/v${version}/${pkgname}-${version}.tar.xz"
-checksum=85f3f935e76b420f2e5e272514368fdfbe4c25c01daa1e161ac28a6e1edb0d2c
+homepage="https://gitlab.com/azelpg/azpainter"
+distfiles="https://gitlab.com/azelpg/azpainter/-/archive/v${version}/azpainter-v${version}.tar.gz"
+checksum=0b724f043246cadacff47885712e5e0cf35ee993c97261b83c4ff1bfd6a80d33
+
+do_configure() {
+	./configure --prefix=/usr
+}
```

**Comment by hervyqa** [comment]
@kruceter great. thank you.. 🎉

File: srcpkgs/azpainter/template [hunk 1]
```diff
@@ -1,15 +1,19 @@
 # Template file for 'azpainter'
 pkgname=azpainter
-version=2.1.6
+version=3.0.6
 revision=1
-build_style=gnu-configure
-hostmakedepends="libpng-devel libjpeg-turbo-devel libXi-devel
- fontconfig-devel"
-makedepends="$hostmakedepends"
+build_style=cmake
```
Comment (classabbyamp) [inline srcpkgs/azpainter/template:5]: let's not abuse the cmake build-style here

**Pull request closed without merging**